core/chains/evm/client/errors.go

```diff
@@ -160,6 +160,19 @@ var harmony = ClientErrors{
 	Fatal:                   harmonyFatal,
 }
 
+// Klaytn
+// https://github.com/klaytn/klaytn/blob/dev/blockchain/error.go
+// https://github.com/klaytn/klaytn/blob/dev/blockchain/tx_pool.go
+var klaytn = ClientErrors{
+	NonceTooLow:                 regexp.MustCompile(`(: |^)nonce too low$`),
+	TransactionAlreadyInMempool: regexp.MustCompile(`(: |^)(known transaction|there is another tx which has the same nonce in the tx pool)`),
```

Comment:
ought it to terminate with `$`?

Reply:
Thank you for the comment. I added `$` at the end of `there is another tx which has the same nonce in the tx pool`. In case of `known transaction`, it can have additional information at the end of the sentence.